sdk-base/src/main/java/com/mapbox/maps/plugin/viewport/ViewportPlugin.kt

```diff
@@ -42,9 +42,10 @@ interface ViewportPlugin : MapPlugin {
    * If transition is canceled, state goes to IDLE.
    *
    * @param targetState The target [ViewportState] to transition to.
+   * @param transition The [ViewportTransition] that's used to transition to target state, if not specified, the [ViewportPlugin.defaultTransition] will be used.
```

Comment:
nit : contains outdated `   * If current status is IDLE, returns ViewportStatus.State(null).` above on the line 31